packages/manager/apps/container/src/core/container/ContainerProvider.jsx

```diff
@@ -31,12 +37,17 @@ export const ContainerProvider = ({ children }) => {
       })
       .then((value) => {
         let version = null;
-        if (value['pnr:betaV1'] === true) {
-          version = BETA_V1;
-        }
-        if (value['pnr:betaV2'] === true) {
-          version = BETA_V2;
+        if (isBetaForced()) {
+          version = getBetaVersionFromLocalStorage();
+        } else {
+          if (value['pnr:betaV1'] === true) {
```

Comment:
***nitpick***

i know this was coded before, but writing `if (value['pnr:betaV1'])` might be better than checking the equality with `true`